app/infrastructure/parsers/lms_parser.py

```
@@ -0,0 +1,301 @@
+
+    def parse_student_information(self, content: BeautifulSoup) -> dict:
+        major = content.find("input", id="id_department")["value"]
+        department, major = DiscriminationUtils.get_department(major=major)
+        return {
+            "name": content.find("input", id="id_firstname")["value"],
+            "college": content.find("input", id="id_institution")["value"],
+            "department": department,
+            "major": major,
+        }
+
+    def parse_course_list(self, content: BeautifulSoup, past: bool) -> list[dict]:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)